Fortnite-API/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"author": {
+		"name": "Leafage",
+		"id": "631499344863821877"
+	},
+	"service": "Fortnite-API",
+	"description": {
+		"en": "Stop paying for outdated API's. We provide a reliable API with always updated content."
+	},
+	"url": [
+		"https://fortnite-api.com",
+		"fortnite-api.com"
+	],
+	"version": "0.1",
+	"logo": "https://cdn.discordapp.com/attachments/634078451841761286/648591677748936704/logo.png",
+	"thumbnail": "https://prnt.sc/q1x3at",

```

Comment:
Upload both logo and thumbnail to a site like imgur and then use the .png link

Reply:
K, i'll change them now

Reply:
Changed